.github/workflows/ci.yaml

```diff
@@ -47,15 +48,15 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        go-version: [1.17.x]
+        go-version: [stable, oldstable]
```

Comment:
should we keep a `oldest` version in the matrix you think? (could be the `1.17.x` or perhaps slightly newer if we want the updated `go.mod` format (which I think was in go1.18, not 100% sure)

Reply:
done